app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ArticlesController < ApplicationController
+
+  def index
+    @articles = Article.where(delete_flg: false).order(created_at: :desc)
+    # activeスコープで定義しているから上と同じ意味になる↓
+    # @articles = Article.active.order(created_at: :desc)
+  end
+
+  def about
+  end
+
+  def contact
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def edit
+    # パラメータ検証必要
+    return redirect_to articles_path if params[:id].to_i <= 0
+    @article = Article.find_by(id: params[:id])
+  end
+
+  def create
+    #render plain: params[:article].inspect
+    #raise params.inspect
+    @article = Article.new(article_params)
+    if @article.save
+      redirect_to articles_path
+    else
+      render "new"
```

Comment:
これでもOKですが、アクション名は文字列ではなくシンボルで書くことが多いですね。
```suggestion
      render :new
```

Reply:
かしこまりました！勉強になります。